docs/tutorials/hostpath/xfs_quota/modify-xfs-quota.md

```diff
@@ -0,0 +1,106 @@
+# OpenEBS LocalPV Hostpath Modify XFS Quota
+
+Follow these steps to change/remove XFS project quota enforcement from existing hostpath volumes:
+
+## Step 1:
+
+Make a note of the BasePath directory used for the hostpath volume. The default BasePath is '/var/openebs/local'. You can get the BasePath set on the StorageClass by executing the following command:
+```console
+$ kubectl describe sc <storageclass-name>
+```
+
+## Step 2:
+
+Log in to the node where the volume exists. You can find the nodename (or any other identifying label selector) by describing the PV resource.
+```console
+$ kubectl get pvc --namespace demo
+
+NAME              STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS       AGE
+demo-vol-demo-0   Bound    pvc-0365904e-0add-45ec-9b4e-f4080929d6cd   2Gi        RWO            openebs-hostpath   21s
+
+
+$ kubectl describe pv pvc-0365904e-0add-45ec-9b4e-f4080929d6cd
+
+Name:              pvc-0365904e-0add-45ec-9b4e-f4080929d6cd
+Labels:            openebs.io/cas-type=local-hostpath
+Annotations:       pv.kubernetes.io/provisioned-by: openebs.io/local
+Finalizers:        [kubernetes.io/pv-protection]
+StorageClass:      openebs-hostpath
+Status:            Bound
+Claim:             demo/demo-vol-demo-0
+Reclaim Policy:    Delete
+Access Modes:      RWO
+VolumeMode:        Filesystem
+Capacity:          2Gi
+Node Affinity:     
+  Required Terms:  
+    Term 0:        kubernetes.io/hostname in [storage-node-2]
+Message:           
+Source:
+    Type:  LocalVolume (a persistent volume backed by local storage on a node)
+    Path:  /var/openebs/local/pvc-0365904e-0add-45ec-9b4e-f4080929d6cd
+Events:    <none>
+
+
+$ kubectl get node -l 'kubernetes.io/hostname in (storage-node-2)'
+
+NAME             STATUS   ROLES    AGE   VERSION
+storage-node-2   Ready    worker   10m   v1.22.1
+```
+
+## Step 3:
+
+You can change the soft and/or hard limit of an existing hostpath volume with XFS project quota enabled by following the steps below. If you'd like to remove the XFS project quota entirely, move on to the [Remove project](#remove-project) sub-section.
+
+### Change limits
+
+Execute the following commands on the node where the hostpath volume exists.
+
+Make a note of the Project ID.
+```command
+$ sudo xfs_quota -x -c 'report -h' /var/openebs/local
+
+Project quota on /var/openebs/local (/dev/nvme1n1)
+                        Blocks              
+Project ID   Used   Soft   Hard Warn/Grace   
+---------- --------------------------------- 
+#0              0      0      0  00 [------]
+#1             1G   2.0G   2.0G  00 [------]
+```
+
+Modify the limits as desired using the following command. The values of bhard and bsoft must be in KB (not KiB). The sample command below sets the bsoft and bhard values to 0 for a project ID=1.
```

Comment:
Not necessary to be only in KB. It can be in any unit K/M/G/T.. except it cannot in KiB/MiB...

Reply:
I hade made a syntax error while testing, and assumed it was because of the units. Thank you for pointing this one out.